logic/transaction_pool.js

```diff
@@ -787,14 +795,61 @@ __private.isTransactionInUnconfirmedQueue = function(transaction) {
 	return typeof self.unconfirmed.index[transaction.id] === 'number';
 };
 
+/**
+ * Check if transaction is from multisig account.
+ *
+ * @private
+ * @param {Object} transaction - Transaction object
+ * @returns {Boolean}
+ */
+__private.isMultisigTransaction = function(transaction, cb) {
+	let hasSender;
+
+	async.waterfall(
+		[
+			function getAccount(waterCb) {
+				if (transaction.senderPublicKey) {
+					hasSender = true;
+					modules.accounts.getAccount(
+						{ publicKey: transaction.senderPublicKey },
+						waterCb
+					);
+				} else {
```

Comment:
Sweetie, you need to call `waterCb` in `else` block also, or Jenkins will be stuck for hours when hit that condition (like it happening currently)... 😢